README.md

```diff
@@ -1,33 +1,25 @@
 # Random Spotify Song
 
-This project contains Python files that will play a random song from Spotify by looking up the top thirty songs for a random three-letter string or commonly used word and picking one of them.
+## How to use:
 
-In order to allow the program to authenticate with Spotify, you will need to install the `spotipy` module for Python. You can install the latest version with the command:
+This project allows you to play a random song from Spotify by looking up the top thirty songs for a random three-letter string (char schema) or commonly used word and picking one of them (word schema).
 
-```
-pip install git+https://github.com/plamere/spotipy.git --upgrade
-```
+To run the program, double click on `RandomSpotifySong.bat`. There is also a shortcut file which can be moved from the original folder and runs the same as `RandomSpotifySong.bat`.
 
-The first time you instantiate `RandomPlayer`, it will prompt you for your username (and store that value locally) and then it should redirect you to a web browser and authenticate with Spotify. Once authenticated, you will be redirected to a `localhost` address that will probably not load. You should paste the url you are redirected to into the terminal. This authenticates the program with Spotify.
+The first time you run the program, it will prompt you for your username (and store that value locally) and then it should redirect you to a web browser and authenticate with Spotify. Once authenticated, you will be redirected to a `localhost` address that will probably not load. You should paste the url you are redirected to into the terminal. This authenticates the program with Spotify.
+
+## For use in other python programs:
 
 The `RandomPlayer` class works is designed to work as per the following template:
 
 ```Python
 player = RandomPlayer(random_song_schema='word') # random_song_schema is either 'word' or 'char' depending on how you want to generate random songs (see above)
 player.authenticate()
-
 player.playRandomSong(numSongs = 10) # Will throw an AuthenticationError if player isn't authenticated
 ```
 
 That code would start playback of 10 random songs on Spotify.
 
-You can also do the same thing by executing the following command:
```

Comment:
I think it'd be useful to keep some instructions about running the file with the `python` executable just to keep it OS-independent. Thoughts?